### PR TITLE
Fixes CSV parsing in shadowtau

### LINKED ISSUE #5315: failed to convert string [] to a double
**ISIS version(s) affected**: x.y.z

**Description**  

I use the "shadowtau" for HRSC to estimate tau.I have exactly follow the instruction from https://isis.astrogeology.usgs.gov/7.0.0/Application/presentation/Tabbed/shadowtau/shadowtau.html . However  it reports this error -- failed to convert string to a double.
image
I want to konw what cause this and hhow to  slove it ? Thanks a lot.
Here is my basic environment:
image

**How to reproduce**  

**Possible Solution**  

**Additional context**

### PR DESCRIPTION
**Pull request by acpaquette**
## Description
The shadowtau app in ISIS was failing due to the csv parsing using two delimiters at once, this has been fixed and functionality remains the same.

## Related Issue
Fixes #5315

## How Has This Been Validated?
Validated by hand and in current tests, more tests should be added to directly verify failure cases

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Infrastructure change (changes to things like CI or the build system that do not impact users)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/Code-Of-Conduct.md)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [x] I have added any user impacting changes to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

**Review by chkim-usgs — DISMISSED** [review]
Was able to manually test the original issue and trigger the new error messages, looks good.

File: isis/src/base/apps/shadowtau/main.cpp [hunk 1]
```diff
@@ -250,17 +250,45 @@ void IsisMain() {
   // rejected if there aren't enough words or if any of them don't make
   // sense as the corresponding parameter.
   FileName sInFileName(sInFile);
-  TextFile infile(sInFileName.expanded());
+  // Try with the default ',' delimieter, if that only produces one row
```
Comment (AustinSanders) [inline isis/src/base/apps/shadowtau/main.cpp]: delimieter -> delimiter

**Review by AustinSanders — CHANGES_REQUESTED** [review]
quick fix on comment, otherwise looks good

File: isis/src/base/apps/shadowtau/main.cpp (hunk 1 above)
Comment (AustinSanders) [reply isis/src/base/apps/shadowtau/main.cpp]: Also needs changelog entry

**Review by AustinSanders — APPROVED** [review]

**Pull request merged**